test/Table_Tests/src/In_Memory/Table_Spec.enso

```diff
@@ -188,7 +188,8 @@ add_specs suite_builder =
 
             (table.at "enso_dates" == table.at "py_dates").to_vector . should_equal [True, True]
 
-        group_builder.specify "should work with polyglot values coming from JS" <|
+        pending_js_missing = if Polyglot.is_language_installed "js" then Nothing else "Can't run JavaScript tests, language `js` is not installed."
+        group_builder.specify "should work with polyglot values coming from JS" pending=pending_js_missing <|
```

Comment:
I see. JavaScript is no longer in the `bin/enso` and thus these tests have to be pending.